satellite/payments/account.go

```diff
@@ -0,0 +1,18 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package payments
+
+import (
+	"github.com/skyrings/skyring-common/tools/uuid"
+)
+
+// Accounts exposes all needed functionality to manage payment accounts
+type Accounts interface {
+	Setup(userID uuid.UUID) error
+}
+
+// Account stores all payment related information
+type Account struct {
+	Balance float64
```

Comment:
Does `float64` have sufficient precision?

Reply:
We should use `int64` for amount in cents, since it cannot get smaller than that

Reply:
The API documents in https://www.coinpayments.net/apidoc-code and https://www.coinpayments.net/apidoc-rates show values like `0.018343387500000000000000` and `1.000000000000000000000000`. I assume this could be for all amounts.

Reply:
That is fiat currency balance, USD only for now. Applied tokens/credits already converted at this stage

Reply:
It also shows `"tx_fee": "0.00100000",` and here https://www.coinpayments.net/apidoc-create-transaction the amount is as `"amount":"1.00000000"`.

Reply:
Fee is calculated in received token currency. Yeah I understand what you are saying, but that is coinpayments implementation side, while here we only show balance in USD, which stored on the `stripe` side

Reply:
Also that struct should be deleted as it is scope of different PR

Reply:
If we don't get problems with fractions, then sure. Just making sure.